### ISSUE TITLE
[Feature Request] how many users can ampache handle?

### ISSUE THREAD

**Issue by izyspania** [ampache/ampache#3114]
How many users can ampache handle? What specs should server have for about 500 users? Im building and android app (subsonic api) with audiobooks for the blind people in my city (city association) so it could have from 100 to 2000-3000 members and i am not sure what server hardware should i have for this many users (im just assuming that sometimes they could be like 500 users at the same time , prob less but to be on the safe side) or if ampache is suited for this amount of users. Most of them they will use the android app so the most traffic/usage with be from that. Thanks

**Comment by lusum** [comment]
Btw, really interesting idea, i would like to use an audiobook reader with ampache.

**Comment by izyspania** [comment]
Yes , it would be nice if it had import and a template for audiobooks (or maybe a plugin?)

**Comment by lachlan-00** [comment]
I have a little bit under 400k files on my server with about 12 users. Ampache will scale up or down pretty well it's really just hardware requirements.

I always recommend Apache2 with php-fpm over nginx but that's just me.

You can have multiple webservers connecting to mariadb/mysql and keep a cache of art on the webserver using an nfs share
![image](https://user-images.githubusercontent.com/1305249/145134095-172cf761-286c-4a5d-8c8a-1ae546e3695f.png)

I pre-cache all my flac and wav files into mp3 using transcode caching which speeds up the process if you want different bitrates. This can then be cached on each webserver similar to art
https://github.com/ampache/ampache/wiki/Transcode-Caching

You're really just limited by how much effort, hardware and time you want to throw at it,

RE: Audiobooks
My intention is to make Audiobooks like Podcasts where every podcast is a book and the episodes translate to files/chapters.

I just have to get to it at some point.

**Comment by izyspania** [comment]
Thanks for the info
